src/storage/lock_manager/mod.rs

```diff
@@ -124,31 +180,68 @@ pub trait LockManager: Clone + Send + 'static {
 
 // For test
 #[derive(Clone)]
-pub struct DummyLockManager;
+pub struct DummyLockManager {
```

Comment:
Since it has some functionality now, do you think it more appropriate to call it a `MockLockManager` instead of `Dummy`?